services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationController.kt

```diff
@@ -269,6 +276,15 @@ class ApplicationController(
         }
     }
 
+    fun checkApplicationPermission(hankeTunnus: String?, userId: String) {
+        hankeTunnus.let { tunnus ->
```

Comment:
This `let` doesn't do anything here, just aliases `hankeTunnus` with `tunnus`.

Reply:
This is now irrelevant as we kept Application hanketunnus non nullable. Changed